### DISCUSSION TITLE
What is specific about it?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#744] [Galaxy Trivia] [answered]
6th of May, 1978
12:34 PM

**Comment by SANKARAMDAS** [comment, marked as answer]
This one took a few minutes.
12:34 5/6/78
1,2,3,4,5,6,7,8

On this day at 12:34, the numbers 12345678 represented the time and day: 12:34 5/6/78. The next such sequence will occur in 2078

**Reply by ParanoidUser** [reply]
Oops, you did it again... congratulations! 👏👏 You have just earned the prestigious [🥈 Galaxy Brain](https://github.com/SANKARAMDAS?tab=achievements&achievement=galaxy-brain) badge. Keep up the great work and unlock the Golden 🥇 badge next by answering 32 questions. Don't forget to post your own questions too! 😊🎉

**Comment by peterbaccaro** [comment]
At that moment, the time and day could be written as: **12:34, 5/6/78** (where the numbers 1 to 8 appear in sequence).
